lib/config/types.ts

```diff
@@ -260,6 +260,8 @@ export interface RenovateConfig
 
   checkedBranches?: string[];
   customizeDashboard?: Record<string, string>;
+
+  statusCheckNames?: Record<string, string>;
```

Comment:
can we limit key to known strings and us that inside validation?

that way we can also validate key typos at compile time